benchmark/README.md

```diff
@@ -1204,3 +1204,159 @@ cooling_down = 0
+          vgg16_int8  min =  860.20  max =  888.42  avg =  876.37
+            resnet50  min =  242.42  max =  246.66  avg =  244.09
+       resnet50_int8  min =  495.75  max =  510.46  avg =  504.18
+      squeezenet_ssd  min =   67.77  max =   83.61  avg =   71.78
+ squeezenet_ssd_int8  min =  127.00  max =  145.89  avg =  135.29
+       mobilenet_ssd  min =  102.18  max =  105.09  avg =  103.47
+  mobilenet_ssd_int8  min =  216.56  max =  222.43  avg =  219.49
+      mobilenet_yolo  min =  234.75  max =  260.54  avg =  246.26
+  mobilenetv2_yolov3  min =  117.52  max =  119.43  avg =  118.24
+         yolov4-tiny  min =  136.97  max =  140.14  avg =  138.56
+
+
+$ ./benchncnn 8 1 2 0 1
+[0 NVIDIA Tegra Xavier (nvgpu)]  queueC=2[8]  queueG=0[16]  queueT=1[1]
+[0 NVIDIA Tegra Xavier (nvgpu)]  bugsbn1=0  buglbia=0  bugihfa=0
+[0 NVIDIA Tegra Xavier (nvgpu)]  fp16p=1  fp16s=1  fp16a=1  int8s=1  int8a=1
+loop_count = 8
+num_threads = 1
+powersave = 2
+gpu_device = 0
+cooling_down = 1
+          squeezenet  min =    3.46  max =   15.43  avg =    6.67
+     squeezenet_int8  min =   57.20  max =   59.36  avg =   58.11
+           mobilenet  min =    3.55  max =   18.06  avg =   10.02
+      mobilenet_int8  min =  128.38  max =  130.48  avg =  129.51
+        mobilenet_v2  min =    4.81  max =   17.19  avg =    9.51
+        mobilenet_v3  min =    6.02  max =   20.60  avg =   13.07
+          shufflenet  min =    4.28  max =   17.97  avg =    8.85
+       shufflenet_v2  min =    4.27  max =   41.05  avg =   20.22
+             mnasnet  min =    4.70  max =   16.69  avg =    9.52
+     proxylessnasnet  min =    4.63  max =   16.76  avg =    6.27
+     efficientnet_b0  min =   10.32  max =   24.72  avg =   18.81
+        regnety_400m  min =    6.44  max =   35.53  avg =   15.08
+           blazeface  min =    2.32  max =   13.39  avg =    4.44
+           googlenet  min =   10.53  max =   37.23  avg =   21.88
+      googlenet_int8  min =  196.47  max =  201.26  avg =  197.52
+            resnet18  min =    6.61  max =   34.69  avg =   15.42
+       resnet18_int8  min =  159.57  max =  168.56  avg =  163.05
+             alexnet  min =    8.06  max =   22.83  avg =   15.57
+               vgg16  min =   32.74  max =   89.11  avg =   64.80
+          vgg16_int8  min =  872.47  max =  886.24  avg =  880.69
+            resnet50  min =   13.92  max =   40.74  avg =   30.52
+       resnet50_int8  min =  466.06  max =  476.91  avg =  470.40
+      squeezenet_ssd  min =   11.53  max =   38.27  avg =   18.68
+ squeezenet_ssd_int8  min =  129.05  max =  135.18  avg =  131.57
+       mobilenet_ssd  min =    7.69  max =   34.02  avg =   19.65
+  mobilenet_ssd_int8  min =  214.68  max =  219.59  avg =  217.03
+      mobilenet_yolo  min =   11.40  max =   39.92  avg =   24.83
+  mobilenetv2_yolov3  min =    7.70  max =   25.49  avg =   15.46
+         yolov4-tiny  min =   11.28  max =   34.75  avg =   22.27
```

Comment:
The results here is very unstable, is there another process using the gpu? please retest and confirm it.

Reply:
Maybe another process using the gpuI. I will retest and confirm it.

Reply:
The new test result is updated